gate/gate-web/src/test/resources/gate-test.yml

```diff
@@ -34,6 +34,10 @@ services:
 slack:
   baseUrl: "https://slack.com"
 
+logging:
```

Comment:
Can you move this OAuth2IntegrationWithWireMockTest (and other classes that need it)?  Lots of tests use gate-test.yml but don't need such detailed security logging.